packages/runtime/src/decorators/response.ts

```diff
@@ -1,10 +1,18 @@
+import { HttpStatusCodeLiteral, HttpStatusCodeLiteralStr } from '../interfaces/response'
+
 export function SuccessResponse(name: string | number, description?: string): Function {
   return () => {
     return;
   };
 }
 
-export function Response<T>(name: string | number, description?: string, example?: T): Function {
+export function Response<T>(
+  // The status code should not be allowed to be 200 or "200"` since that return type is already defined by the return of the route function.
+  // @see https://github.com/lukeautry/tsoa/issues/119
+  name: Exclude<HttpStatusCodeLiteral, 200> | Exclude<HttpStatusCodeLiteralStr, '200'>,
```

Comment:
This does not work. OpenAPI allows for strings like `'default' | '4xx'` etc. This would break compatibility.

Reply:
I see. I will change it to `Exclude<HttpStatusCodeLiteral, 200> | Exclude<string, '200'>` in the next commit. WDYT?